### PR TITLE
Improve the documentation and added save query button

### PR DESCRIPTION
**Pull request by leogregianin**
Hi,
I made some changes to improve the documentation to run on Windows and added button "Save Query" in multiline mode to create a text file recording the query.
Best regards, Leonardo.

### CODE REVIEW

File: relational-windows.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+cd relational_gui
+pyuic5 -o maingui.py maingui.ui
+pyuic5 -o survey.py survey.ui
+pyuic5 -o rel_edit.py rel_edit.ui
+
```
Comment (ltworf) [inline relational-windows.bat]: Doesn't create the `.py` file from the resource file.

File: relational_gui/guihandler.py [hunk 2]
```diff
@@ -175,10 +175,10 @@ def optimize(self):
                 self.user_interface.relations,
                 debug=trace
             )
-            print('==== Optimization steps ====')
-            print(query)
-            print('\n'.join(trace))
-            print('========')
+            #print('==== Optimization steps ====')
```
Comment (ltworf) [inline relational_gui/guihandler.py]: Eh, might as well remove the lines then, no?

**Comment by ltworf** [comment]
Hello,

thanks for the changes :)

I have commented a couple of things that should be addressed before merging this.

I don't know if a button to save the query makes much sense. The UI is quite crowded as it is, so I'd probably leave that comment out.

For the future, I would rather you send me by email the patch that I can just merge with git, rather than the github pull request thing, so it is easier for me to try it and merge it.

**Comment by ltworf** [comment]
Do you plan to address my comments or can I ignore this?

File: relational-windows.bat (hunk 1 above)
Comment (ltworf) [inline relational-windows.bat]: It also doesn't do the replacing text thing that the makefile does.

**Comment by leogregianin** [comment]
Sorry about the delay in the response. The option to save the query was interesting in the classroom. Maybe you could put it in the "File" menu. It's your decision.

**Pull request closed without merging**